jupyter-book/linear_models/linear_models_module_intro.md

```diff
@@ -4,10 +4,14 @@
 
 <!-- Give in plain English what the module is about -->
 
-This module will present more in details models using a linear parametrization.
+In this module, will go further into details regarding models that use 
+linear parametrization.
+\
```

Comment:
Hmmm those are real `\` ... can I ask why, maybe a quirk of your editor, maybe a subtle markdown thing? 

Unless there is a good reason I am missing, I think they should be removed (here and in the other files)

Reply:
Hi

It's a subtle markdown thing ;)

It allows to have line breaks without space between two lines, and is used to break down massive paragraphs.

For instance:

This paragraph uses the anti-slash trick,\
and while there is a line break, there is no extra-space in-between.

```
This paragraph uses the anti-slash trick,\
and while there is a line break, there is no extra-space in-between.
```

Another way of doing it: using double spaces at the end of the line, but it is subtle.  
See https://gist.github.com/shaunlebron/746476e6e7a4d698b373

While it is not part of the Markdown specifications, it is included in most markdown readers.

Reply:
Interesting I did not know this. I would favor a blank line, this seems so much simpler ...

Reply:
<rant>Also I am not a big fan of back-slashes in general</rant> :smirk:

Reply:
(maybe because I had to work on Windows at one point, I don't know ...)

Reply:
you can just rewrap (single break line). I think Python people don't like backslash :)